### ISSUE TITLE
Design pattern for transport

### ISSUE THREAD

**Issue by pgaudet** [geneontology/design_patterns#9]
Hi @dosumis 

Right now the design pattern for [COMPOUND X] TRANSPORT	
is 
"transport
 and ('transports or maintains localization of' some [COMPOUND X])" 

Would it work if we used 'has participant' instead of 'transports or maintains localization of' ? (in protégé is seems OK - by doing some tests on a small sample). 

The advantage is that this is would more consistent with the GO structure, because 'maintenance of location' is a sibling of 'establishment of localization'.

I think logically it makes no difference, but using a relation that contains "or" ('transports or maintains localization of') for the specific processes is not very intuitive. 

Thanks, Pascale

**Comment by dosumis** [comment]
> Would it work if we used 'has participant' instead 

No.  You need to keep open the possibility of transport processes having a participant that is not the cargo.  e.g.  a vesicle that transports something or a motor protein.   Bear in mind that if something participates in part of a process, it participates in the process, so arguably, vesicle docking protein are participants in transport.  I think we need a general reference document for the participant branch of the relations ontology.

> The advantage is that this is would more consistent with the GO structure, because 'maintenance of location' is a sibling of 'establishment of localization'.

The structure of GO is a bit of a mess - one reason being that it  confuses localization and location.    I wrote up quite a lot on this problem a couple of years ago, but concluded it was too hard to fix.  I can try to dig my notes out if you're interested. 

> using a relation that contains "or" ('transports or maintains localization of') for the specific processes is not very intuitive.

I agree.  I'd rather just have 'transport'.  But it's hard to fix this, given the structure of the ontology.   I went various different ways of structuring and documented some of the problems, but ended up being conservative and sticking with the original relation.

**Comment by pgaudet** [comment]
Hi David, 

If you find what you wrote, I would very much like to read it, if only to make sure any change I might make does not fall into the issues you've identified. 

Thanks, 
Pascale
